app/src/main/java/com/google/droidjump/GameConstants.java

```diff
@@ -22,4 +22,5 @@
     int DROID_FIRST_STEP_INDEX = 5; // Index of droid at res/mipmap/droid.png image
     int DROID_SECOND_STEP_INDEX = 6; // Index of droid at res/mipmap/droid.png image
     int SLEEP_TIME = 15; // Sleep time in gameView (ms)
+    int NEW_INTERVAL_TIME_POINT =  0; // Beginning of the new time interval between obstacles
```

Comment:
nit: INTERVAL_START_TIME? or something like that?

Reply:
Done